instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/OpenTelemetryConnection.java

```diff
@@ -48,45 +50,52 @@ public class OpenTelemetryConnection implements Connection {
 
   private final Connection delegate;
   private final DbInfo dbInfo;
+  private final OpenTelemetry openTelemetry;
 
+  @Deprecated
```

Comment:
`OpenTelemetryConnection` is an internal class (see the package name and the class javadoc), you don't need to care about deprecating internal methods; you can just remove them.